test/common/network/matching/inputs_integration_test.cc

```diff
@@ -59,7 +87,8 @@ TEST_F(InputsIntegrationTest, DestinationIPInput) {
   initialize("DestinationIPInput", "127.0.0.1");
 
   Network::MockConnectionSocket socket;
-  MatchingDataImpl data(socket);
+  StreamInfo::FilterStateImpl filter_state(StreamInfo::FilterState::LifeSpan::FilterChain);
```

Comment:
FilterChain is meant for HTTP, Connection is more appropriate here.

Reply:
done